bin/start_postgresdb.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-sudo -u postgres PGDATA=/var/lib/postgresql/data /usr/lib/postgresql/13/bin/pg_ctl start
+sudo -u postgres PGDATA=/var/lib/postgresql/data /usr/lib/postgresql/"${POSTGRES_VERSION}"/bin/pg_ctl start
```

Comment:
Not sure why we hadn't already done this? But the env var does live on in any container so this works.